packages/ingest/granule.js

```diff
@@ -469,8 +469,10 @@ class Granule {
    * @returns {Array<Object>} returns the staged file and the renamed existing duplicates if any
    */
   async ingestFile(file, bucket, duplicateHandling) {
+    // place files in the <collectionId> subdirectory
+    const stagingPath = path.join(this.fileStagingDir, this.collectionId);
```

Comment:
This is failing in unit tests because they call `ingestFile` directly meaning `collectionId` is undefined as it is only defined in `ingest()`.